java/Glacier2/greeter/.gitignore

```diff
@@ -0,0 +1,5 @@
+bin
+build
+.gradle
+
+.DS_Store
```

Comment:
I would not include .DS_Store. Or if we do, it would be in the top-level .gitignore.

> A .DS_Store file is a hidden file created by macOS Finder to store custom attributes and metadata about a folder, such as icon positions, view settings, and Spotlight comments. These files are automatically generated when you open a folder with Finder, even on remote file systems. While primarily used by Finder, they can cause issues with backups, version control systems, and folder archives. 

Try:
```
find . -name .gitignore | xargs fgrep DS_Store
```